apps/src/code-studio/pd/workshop_dashboard/legacy_survey_summaries.jsx

```diff
@@ -0,0 +1,96 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Spinner from '../components/spinner';
+
+class LegacySurveySummaryTable extends React.Component {
+  static propTypes = {
+    title: PropTypes.string.isRequired,
+    data: PropTypes.object
+  };
+
+  render() {
+    return (
+      <div>
+        <h2>{this.props.title}</h2>
+        <table>
+          <thead>
+            <tr>
+              <th>Question</th>
+              <th>Score</th>
+            </tr>
+          </thead>
+          <tbody>
+            {Object.keys(this.props.data).map((key, index) => (
+              <tr key={index}>
+                <td>{key}</td>
+                <td>{this.props.data[key]}</td>
+              </tr>
+            ))}
+          </tbody>
+        </table>
+      </div>
+    );
+  }
+}
+
+export default class LegacySurveySummaries extends React.Component {
+  state = {
+    loading: true,
+    data: null
+  };
+
+  componentDidMount() {
+    this.load();
+  }
+
+  load = () => {
+    this.setState({loading: true});
+
+    const queryUrl = '/api/v1/pd/legacy_survey_summaries';
+
+    this.loadRequest = $.ajax({
+      method: 'GET',
+      url: queryUrl,
+      dataType: 'json'
+    }).done(data => {
+      this.setState({
+        loading: false,
+        data: data
+      });
+    });
+  };
+
+  render() {
+    if (this.state.loading) {
+      return (
+        <div>
+          <Spinner />
+        </div>
+      );
+    }
+
+    return (
+      <div>
+        <LegacySurveySummaryTable
+          title="CSF Intro post-workshop"
```

Comment:
nit: to make clear this is for the facilitators workshops, maybe change title to "CSF Intro post-workshop for my workshops" or something similar